modules/distr-tracing-tempo-install-tempomonolithic-web-console.adoc

```diff
@@ -0,0 +1,77 @@
+//
+//* observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="distr-tracing-tempo-install-tempomonolithic-web-console_{context}"]
+= Installing a TempoMonolithic instance by using the web console
+
+:FeatureName: The TempoMonolithic instance
+include::snippets/technology-preview.adoc[]
+
+You can install a TempoMonolithic instance from the *Administrator* view of the web console.
+
+.Prerequisites
+
+* You are logged in to the {product-title} web console as a cluster administrator with the `cluster-admin` role.
+
+* For {product-dedicated}, you must be logged in using an account with the `dedicated-admin` role.
+
+.Procedure
+
+. Go to *Home* -> *Projects* -> *Create Project* to create a project of your choice for the *TempoMonolithic* instance that you will create in a subsequent step.
+
+. Decide which type of supported storage to use for storing traces: in-memory storage, a Persistent Volume, or object storage.
++
+[IMPORTANT]
+====
+Object storage is not included with the {TempoShortName} and requires setting up an object store by a supported provider: link:https://www.redhat.com/en/technologies/cloud-computing/openshift-data-foundation[Red Hat OpenShift Data Foundation], link:https://min.io/[MinIO], link:https://aws.amazon.com/s3/[Amazon S3], link:https://azure.microsoft.com/en-us/products/storage/blobs/[Azure Blob Storage], or link:https://cloud.google.com/storage/[Google Cloud Storage].
+
+Additionally, opting for object storage requires creating a secret for your object storage bucket in the project that you created for the *TempoMonolithic* instance. You can do this in *Workloads* -> *Secrets* -> *Create* -> *From YAML*.
+
+For more information, see "Object storage setup".
+
+--
+include::snippets/distr-tracing-tempo-secret-example.adoc[]
+--
+====
+
+. Create a *TempoMonolithic* instance.
++
+[NOTE]
+====
+You can create multiple *TempoMonolithic* instances in separate projects on the same cluster.
+====
+
+.. Go to *Operators* -> *Installed Operators*.
+
+.. Select *TempoMonolithic* -> *Create TempoMonolithic* -> *YAML view*.
+
+.. In the *YAML view*, customize the `TempoMonolithic` custom resource (CR).
++
+include::snippets/distr-tracing-tempo-tempomonolithic-custom-resource.adoc[]
+
+.. Select *Create*.
+
+.Verification
+
+. Use the *Project:* dropdown list to select the project of the *TempoMonolithic* instance.
+
+. Go to *Operators* -> *Installed Operators* to verify that the *Status* of the *TempoMonolithic* instance is *Condition: Ready*.
+
+. Go to *Workloads* -> *Pods* to verify that the pod of the *TempoMonolithic* instance is running.
+
+. Access the Jaeger UI:
+
+.. Go to *Networking* -> *Routes* and kbd:[Ctrl+F] to search for `jaegerui`.
++
+[NOTE]
+====
+The Jaeger UI uses the `tempo-<metadata_name_of_TempoMonolithic_CR>-jaegerui` route.
+====
+
+.. In the *Location* column, open the URL to access the Jaeger UI.
```

Comment:
```suggestion
.. In the *Location* column, open the URL to access the Jaeger UI:
```

Reply:
There are no more substeps. This is the last substep.

Reply:
My bad! Not sure why those 2 tricked me.